import-service/handlers/importFileParser.ts

```diff
@@ -0,0 +1,51 @@
+import { S3Handler, S3Event, S3EventRecord } from 'aws-lambda'
+import * as AWS from 'aws-sdk'
+import * as csv from 'csv-parser'
+
+import { toSuccess, toError } from '../../core/response'
+import { Bucket } from './const'
+
+// @ts-ignore
+export const importFileParser: S3Handler = (event: S3Event) => {
+    console.log('Incoming Event', event)
+
+    const s3 = new AWS.S3({ region: 'eu-west-1', signatureVersion: 'v4' })
+    if (!event.Records) {
+        return toError('Not Found', { statusCode: 404 })
+    }
+    
+    event.Records
+        .filter((record: S3EventRecord) => record?.s3?.object?.size)
+        .forEach((record: S3EventRecord) => {
+            const key = record.s3.object.key
+            const s3Stream = s3.getObject({
+                Bucket,
+                Key: key
+            }).createReadStream()
+                
+            s3Stream
+                .pipe(csv())
+                .on('data', (data: any) => console.log(data))
+                .on('error', (error) => {
+                    console.log('error', error)
+                    throw error
```

Comment:
Just to be 100% clear: There has never been a guarantee that other handlers will fire if an error is thrown in an event handler.
https://github.com/nodejs/node-v0.x-archive/issues/5114#issuecomment-15308031

 I recommend using [pipeline](https://nodejs.org/api/stream.html#stream_stream_pipeline_source_transforms_destination_callback).